models/constant-baseline/Rupdate-constant-baseline.R

```diff
@@ -0,0 +1,117 @@
+if (baseline == "constant baseline") {
+  # get last observed value for median 
+  
+  last_value <- filtered_observations$value[nrow(filtered_observations)]
+  median <- rep(last_value, num_horizons)
+  
+  # get width
+  sigma <-  filtered_observations %>%
+    dplyr::mutate(difference = c(NA, diff(log(value)))) %>%
+    dplyr::filter(target_end_date > max(target_end_date) - 4 * 7) %>%
+    dplyr::pull(difference) %>%
+    sd()
+  width <- rep(sigma, num_horizons)
+}
+
+
+# Packages ----------------------------------------------------------------
+library(magrittr)
+library(dplyr)
+library(readr)
+library(here)
+library(data.table)
+
+# forecast date -----------------------------------------------------------
+forecast_date <- readRDS(here("data", "target_date.rds"))
+
+sigma <-  filtered_observations %>%
+# Set up functions and data -----------------------------------------------
+source(here::here("utils", "get-us-data.R"))
+source(here::here("models", "timeseries", "utils", "deaths-on-cases-forecast.R"))
+
+# Get data
+weekly_deaths_state <- get_us_deaths(data = "daily") %>%
```

Comment:
does the functionality below already have an argument in `get_us_deaths?

Reply:
I don't think so :( data can either be daily or cumulative in get_us_deaths